api-client/src/endpoints/__tests__/createCategoryChildrenEndpoint.spec.ts

```diff
@@ -18,29 +18,26 @@ describe('createCategoryChildrenEndpoint', () => {
     depth: 1
   }
   const endpoint = createCategoryChildrenEndpoint(baseUrl)
+
   it('should map params to url', () => {
     expect(endpoint.mapParamsToUrl(params)).toEqual(
       `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/children?depth=1&url=${params.cityContentPath}`
     )
   })
+
   it('should map params to url for root category', () => {
     expect(endpoint.mapParamsToUrl({ ...params, cityContentPath: '/augsburg/fa', depth: 0 })).toEqual(
       `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/children?depth=0`
     )
   })
+
   it('should map json to category', () => {
     const children = new CategoriesMapModelBuilder(params.city, params.language).build().toArray()
 
-    ;((mapCategoryJson as unknown) as jest.Mock<
-      (json: JsonCategoryType, basePath: string) => CategoryModel
-      // @ts-ignore
-    >).mockImplementation((json: string) => {
-      if (json === 'myFirstCategory') {
-        return children[0]
-      }
+    ;((mapCategoryJson as unknown) as jest.Mock)
```

Comment:
Here is a weird semicolon, is this some weird ts syntax?

Reply:
Yes, as otherwise typescript will evaluate the bracket as function call or sth like that. The semicolon asserts that the previous expression is finished and has nothing to do with the brackets.

Reply:
The alternative would be:
```suggestion
    const mockMapCategoryJson = ((mapCategoryJson as unknown) as jest.Mock)
    mockMapCategoryJson.mockImplementation()
```

Reply:
But becomes with redundant with the `mocked` utility by @Taggotty.